src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
 [
   {
     "fullName": "Armanul Ambia",
+    "picture": "armanul-ambia.webp",
     "positions": {
-      "S21": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F21": {
-        "title": "Algo Director",
-        "tier": 14
-      },
-      "S22": {
-        "title": "Algo President",
-        "tier": 14
-      },
-      "F22": {
-        "title": "Algo Team Lead",
-        "tier": 14
-      }
-    },
-    "picture": "armanul-ambia.webp"
+      "S21": { "title": "Dev Officer", "tier": 19 },
+      "F21": { "title": "Algo Director", "tier": 11 },
+      "S22": { "title": "Algo President", "tier": 11 },
+      "F22": { "title": "Algo Team Lead", "tier": 11 }
+    }
   },
   {
     "fullName": "Jason Anthony",
-    "positions": {
-      "S21": {
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "jason-anthony.webp"
+    "picture": "jason-anthony.webp",
+    "positions": { "S21": { "title": "Secretary", "tier": 3 } }
```

Comment:
The updated "tier" for Secretary is '4'

`"Secretary": { "id": 4, "index": 300 },`

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/620/commits/ca60f85521a24ec2b643a1e305918a99674a5c88>.

Reply:
I noticed that several review comments were left on this PR that are similar. Consider the above <https://github.com/EthanThatOneKid/acmcsuf.com/pull/620#discussion_r996553848> a reply to your review comments between <https://github.com/EthanThatOneKid/acmcsuf.com/pull/620#discussion_r996526569> and <https://github.com/EthanThatOneKid/acmcsuf.com/pull/620#discussion_r996533968> (inclusive).